packages/backend/src/shared/rapyd/rapyd.d.ts

```diff
@@ -0,0 +1,123 @@
+interface RapydBusinessDetail {
```

Comment:
maybe break each interface in a separate file and have a folder for `interface`?
it is easier to access what you need this why instead of searching in one giant file - we will need more interfaces for rapyd

Reply:
Well, I think that's most of interfaces needed for rapyd. I need to add some apis but I think I don't need to add more interfaces maybe..

Reply:
I think we will need more interfaces related to payments

Reply:
Ah, I think you're right. So create a index.ts, and interface folder you mean? Looks reasonable to me. 
@raducristianpopa do you agree with this?

Reply:
I'm sure we will need more types regarding the Rapyd integration.
> maybe break each interface in a separate file and have a folder for interface?

I don't fancy creating a file for each interface. I'm ok with keeping them in a declaration files for the moment.
> it is easier to access what you need this why instead of searching in one giant file - we will need more interfaces for rapyd

The "good" way would be to declare them as close as possible to the data they represent.